src/Workspaces/Core/Portable/Workspace/Host/Metadata/AnalyzerAssemblyLoaderOptions.cs

```diff
@@ -1,10 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace Microsoft.CodeAnalysis.Host;
-
-internal readonly struct AnalyzerAssemblyLoaderOptions(bool shadowCopy)
```

Comment:
this didn't really seem to be providing a lot of value - was getting created to pass a single boolean as a parameter.